resources/styles/components/reference-book/page-navigation-controls.less

```diff
@@ -0,0 +1,84 @@
+.reference-book {
+  .page-wrapper > .nav {
+    position: fixed;
+    height: 100%;
+    width: calc(~"50% - " @reference-book-page-width / 2 );
+    top: 50px;
+
+    display: -webkit-flex;
```

Comment:
Hmm, I wonder if it might be time to look at http://lesshat.madebysource.com/ rather than recreate these all over ; ) (great link from @pandafulmanda )